src/components/Table/TableReel.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { HTMLAttributes, useEffect, useRef, useState } from 'react'
+import styled, { css } from 'styled-components'
+
+export type Props = {
+  children: React.ReactNode
+}
```

Comment:
imo： 好みですが、React で型が提供されています

```suggestion
export type Props = PropsWithChildren
```